### DISCUSSION TITLE
Stand Up Monday April 24th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#66] [Main Stand Ups]
```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Finish signup/login
 
**What are your priorities for the next 24 hours?**:
- Deploy auth
- work on service for data analysis

**Blockers**:
 - None

**Shoutouts**:
 -@DobleP

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Made some tickets with @CarlosCRG19 to organize the backend development
 - Start working on internal dataset repository
 
**What are your priorities for the next 24 hours?**:
 - Finished dataset repository logic
 - Deploy database to google cloud
  
**Blockers**:
 - None

**Shoutouts**:
 -@AMLO

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
 - Got access to the Udemy API
 - Set the parameters for the courses
 - Made dataframes with the courses 
 - Learned how to use edx api
**What are your priorities for the next 24 hours?**:
 - Get accepted in the edx api 
 - Compare the Udemy courses with the IBM courses
**Blockers**:
 - What we want to answer in the data analysis
**Shoutouts**:
 -@ken

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - I finished entirely login and signup 's input data validation
 - Started trying to implement the API request structure
 - As a test, tried to migrate the app to actual NextJS 13

**What are your priorities for the next 24 hours?**:
 - Define how are we going to make the API reqs
 - Set up the backend on my computer (probably going to need pp)

**Blockers**:
 - I am blanking on the proper way to do calls on NextJS to an **external** API.

**Shoutouts**:
 - @CarlosCRG19 @BrandonMagana for over-engineering the backend
